src/backend/distributed/citus--8.3-1--8.4-1.sql

```diff
@@ -209,13 +214,20 @@ BEGIN
         RETURNING
             type,
             object_names,
-            object_args
+            object_args,
+            distribution_arg_index,
+            colocation_id
     )
-    INSERT INTO citus.pg_dist_object (classid, objid, objsubid)
+    INSERT INTO citus.pg_dist_object (
```

Comment:
We'll move this function in its own file per #2951, so ignore the comment about moving this into its own function.